examples/nodejs-ts-app/README.md

```diff
@@ -20,6 +20,25 @@ There is an example policy included with the example, see
 npm run build
 ```
 
+## Note to Windows users using a PowerShell terminal 
+
+The way that PowerShell 
+escapes characters is different from Bash. See 
+[this](https://github.com/open-policy-agent/opa/issues/3953) for more 
+information. In order to run this example in Windows, use double 
+quotes when using the opa comand to build:
+
+```
+opa build -t wasm -e "example/hello" ./example.rego && tar xzf ./bundle.tar.gz /policy.wasm
+```
+
+The build script in the scripts section in package.json should be modified to 
+use (escaped) double quotes:
+
+```json
+"build": "opa build -t wasm -e \"example/hello\" ./example.rego && tar xzf ./bundle.tar.gz /policy.wasm",
```

Comment:
🤔 what happens with `-e example/hello`? There should be no need to put quotes around this, not in Bash and not in Powershell...

Reply:
You are right. I just tested it and no quotes works well, at least on Windows with PS. If this works on Bash, there is no need for a special note for Windows users. I can squash these and issue a new one as you suggest. By the way, I just found another issue with the README: the command to run the sample should be "npm start..." instead of "npm run...". So, should the new PR remove the quotes from package.json and correct the README with respect to "npm start..."?